.changeset/nice-cows-shout.md

```diff
@@ -0,0 +1,62 @@
+---
+"@spectrum-css/coachmark": major
+---
+
+# S2 coachmark migration
```

Comment:
When this gets injected into the changelog, it'll be an h4 level so maybe we update these to start at h4?